File: Prince_Tiwari_Assignment1/Reverse_Numb_inc.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Prince_Tiwari_Assignment1
{
    internal class Reverse_Numb_inc
    {
        public static void Main(string[] args)
        {
            int n = Convert.ToInt32(Console.ReadLine());

            for (int i = n; i > 0; i--)
            {
                for (int j = 1; j <= i; j++)
                {
                    Console.Write(j);
                }
                Console.WriteLine();
            }
        }
    }
}

File: Prince_Tiwari_Assignment1_VB/Reverse_Numb_Inc.vb
Public Class Reverse_Numb_Inc
    Public Shared Sub Main(args As String())
        Dim n As Integer = Convert.ToInt32(Console.ReadLine())

        For i As Integer = n To 1 Step -1
            For j As Integer = 1 To i
                Console.Write(j)
            Next j
            Console.WriteLine()
        Next i
    End Sub
End Class
